pkg/iss/iss.go

```diff
@@ -29,11 +30,26 @@ import (
 	"github.com/filecoin-project/mir/pkg/pb/isspb"
 	"github.com/filecoin-project/mir/pkg/pb/messagepb"
 	"github.com/filecoin-project/mir/pkg/pb/requestpb"
-	"github.com/filecoin-project/mir/pkg/pb/statuspb"
 	"github.com/filecoin-project/mir/pkg/serializing"
 	t "github.com/filecoin-project/mir/pkg/types"
 )
 
+// ============================================================
+// Global package variables
+// ============================================================
+
+// Names of modules this protocol depends on.
+// The corresponding modules are expected by ISS to be stored under these keys by the Node.
+var (
+	ownModuleName    t.ModuleID = "iss"
+	netModuleName    t.ModuleID = "net"
+	appModuleName    t.ModuleID = "app"
+	walModuleName    t.ModuleID = "wal"
+	hasherModuleName t.ModuleID = "hasher"
+	cryptoModuleName t.ModuleID = "crypto"
+	timerModuleName  t.ModuleID = "timer"
+)
```

Comment:
Those string literals still appear here and there. Maybe though make these constants exported and use throughout the code base consistently?

Reply:
The literals are mostly used in modules that are going to be phased out for ISS, it's not worth the effort at this point. When the module composition becomes stable, we can consolidate much easier.

Reply:
I think we should avoid global constants and allow, for example, different modules to use different hash implementations. For now, these names are hard-coded for ISS, but in the future, I hope they will come from a config.

Reply:
Yes, I already created an issue (#99) for that yesterday.

Reply:
But, unless I overlooked something (possible), the literals don't appear *within* the iss package. The other occurrences fall in the scope of my [previous comment](https://github.com/filecoin-project/mir/pull/98#discussion_r898951057):
> The literals are mostly used in modules that are going to be phased out for ISS, it's not worth the effort at this point. When the module composition becomes stable, we can consolidate much easier.